remote.go

```diff
@@ -466,6 +466,10 @@ func (wd *remoteWebDriver) SwitchFrame(frame string) error {
 	return wd.voidCommand("/session/%s/frame", params)
 }
 
+func (wd *remoteWebDriver) SwitchFrameParent() error {
+	return wd.voidCommand("/session/%s/frame/parent", nil)
```

Comment:
My interpretation of the code is that `%s` is the session ID. [`/frame/parent`](https://github.com/SeleniumHQ/selenium/wiki/JsonWireProtocol#sessionsessionidframe) doesn't take a frame as a parameter, it just goes to the parent of the current frame.